doc/user/layouts/partials/create-source/envelope/upsert/details.html

```diff
@@ -2,7 +2,14 @@
 
 Specifying `ENVELOPE UPSERT` creates a source that supports Kafka's standard
 key-value convention, and supports inserts, updates, and deletes within
-Materialize. The source is also compatible with Kafka's log-compaction feature.
+Materialize. The source is also compatible with Kafka's log-compaction feature,
+and can be useful for users who want to compact
+[CDC sources](https://materialize.com/docs/third-party/debezium/).
+
+While the upsert envelope format is more powerful than the default (append-only)
```

Comment:
I still disagree with any phrasing that gives a value judgment, like "more powerful" -- the semantics are just _different_, and needed for different types of data.

I'd prefer something like "sometimes necessary", "often useful", or similar.

Reply:
Yeah I agree with Brennan and would suggest something like:

> To correctly handle data sources that do not publish their own retractions the Upsert format
> must maintain state proportional to the number of unique rows in the source, and it must perform
> extra work to generate the implied retractions based on that state. This has makes the Upsert
> format slower and more memory-intensive than other formats.

Reply:
Thanks!  Agreed, I took another pass at this (and just completely dropped the value judgement).  Hopefully this is now resolved